src/ui/components/shared/WorkspaceSettingsModal/utils.tsx

```diff
@@ -81,9 +95,5 @@ export const cycleCharge = (planPricing: SubscriptionWithPricing): number => {
     return 0;
   }
 
-  let monthsPerCycle = 1;
-  if (planPricing.billingSchedule === "annual") {
-    monthsPerCycle = 12;
-  }
-  return monthsPerCycle * planPricing.seatPrice * planPricing.seatCount;
+  return planPricing.seatPrice * planPricing.seatCount;
```

Comment:
this should be `(planPricing.billingSchedule == "annual" ? 12 : 1) *  planPricing.seatPrice * planPricing.seatCount * (1 - planPricing.discount)`